config/locales/idv/fr.yml

```diff
@@ -86,8 +90,13 @@ fr:
     form:
       address1: Adresse
       address2: Adresse (optional)
-      city: Ville une lettre contenant un code.′
```

Comment:
This translation got messed up in [this PR](https://github.com/18F/identity-idp/pull/5300/files#r908898393) -- this restores it to what it should be